model/flow/constants.go

```diff
@@ -42,22 +42,25 @@ const DefaultMaxAddressIndex = 20_000_000
 // badger value log GC. Equivalent to 10 mins for a 1 second block time
 const DefaultValueLogGCFrequency = 10 * 60
 
+// DomainTagLength is set to 32 bytes.
+//
+// Signatures on Flow that needs to be scoped to a certain domain need to
+// have the same length in order to avoid tag collision issues, when prefixing the
+// message to sign.
 const DomainTagLength = 32
 
 const TransactionTagString = "FLOW-V0.0-transaction"
 
-const UserTagString = "FLOW-V0.0-user"
-
```

Comment:
this might be in use by go-sdk or others ?

Reply:
It is in use, this is a breaking change for the `flow-go-sdk/examples`. We should keep that in mind.

Reply:
I'll update the sdk too after merging this PR: https://github.com/onflow/flow-go-sdk/issues/273